modules/ROOT/pages/microsoft-sharepoint-2013-connector.adoc

```diff
@@ -68,25 +80,63 @@ The parameter *_wa_* tells the STS that a signin is being initiated. The *_wtrea
 
-In addition to the basic configuration shown above, there is an advanced setting where you can provide a `Security Token Provider` instance of your own. This mechanism allows you to obtain the security token using a custom implementation and allows the connector refreshing the connection when the token has expired (which is not the case for when just providing the `Security Token` as XML document).
+To obtain a security token, make a POST request to `+https://login.microsoftonline.com/extSTS.srf+` with this XML body:
 
-* *Security Token Provider*: (Optional) A string that references to a Spring bean which should be a class that implements the `org.mule.module.sharepoint.connection.providers.SecurityTokenProvider` interface. Within the implementation you are responsible of providing a security token each time the connector requests one. In order to provide a reference to your own implementation as a Spring bean, refer to the following xref:using-spring-beans-as-flow-components.adoc[documentation].
-
-== To Install this Connector
+[source,text,linenums]
+----
+<s:Envelope xmlns:s="http://www.w3.org/2003/05/soap-envelope"
+      xmlns:a="http://www.w3.org/2005/08/addressing"
+      xmlns:u="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd">
+  <s:Header>
+    <a:Action s:mustUnderstand="1">http://schemas.xmlsoap.org/ws/2005/02/trust/RST/Issue</a:Action>
+    <a:ReplyTo>
+      <a:Address>http://www.w3.org/2005/08/addressing/anonymous</a:Address>
+    </a:ReplyTo>
+    <a:To s:mustUnderstand="1">https://login.microsoftonline.com/extSTS.srf</a:To>
+    <o:Security s:mustUnderstand="1"
+       xmlns:o="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-secext-1.0.xsd">
+      <o:UsernameToken>
+        <o:Username>[username]</o:Username>
+        <o:Password>[password]</o:Password>
+      </o:UsernameToken>
+    </o:Security>
+  </s:Header>
+  <s:Body>
+    <t:RequestSecurityToken xmlns:t="http://schemas.xmlsoap.org/ws/2005/02/trust">
+      <wsp:AppliesTo xmlns:wsp="http://schemas.xmlsoap.org/ws/2004/09/policy">
+        <a:EndpointReference>
+          <a:Address>[endpoint]</a:Address>
+        </a:EndpointReference>
+      </wsp:AppliesTo>
+      <t:KeyType>http://schemas.xmlsoap.org/ws/2005/05/identity/NoProofKey</t:KeyType>
+      <t:RequestType>http://schemas.xmlsoap.org/ws/2005/02/trust/Issue</t:RequestType>
+      <t:TokenType>urn:oasis:names:tc:SAML:1.0:assertion</t:TokenType>
+    </t:RequestSecurityToken>
+  </s:Body>
+</s:Envelope>
+----
+
+The response from the previous request contains the security token between the <wsse:BinarySecurityToken> tags.
+
+After you have a security token, specify the token value in Studio:
+
+image::sharepoint/sharepoint-security-token-config.png[Security Token Configuration]
+
+* Security token: Enter the corresponding security token
+* Site URL: Enter the corresponding site URL
+
+== Install this Connector
```

Comment:
Wasn't this "Install the Connector" in the previous two PRs? 
"This" takes a cap. in titles.